packages/components/divider/src/divider.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <hr
+    v-if="props.orientation === PuikDividerOrientation.HORIZONTAL"
+    class="puik-divider puik-divider--horizontal"
+    :data-test="dataTest ?? undefined"
```

Comment:
```suggestion
    :data-test="dataTest"
```

La valeur par defaut est deja `undefined` donc pas besoin de ce check ;D
Pareil pour la ligne 10

Reply:
yes ! done @MikeLebeau